mira/modeling/ops.py

```diff
@@ -0,0 +1,70 @@
+"""Operations for template models."""
+
+import itertools as itt
+from typing import Iterable, Optional, Set, Tuple, Type
+
+from ..metamodel import ControlledConversion, NaturalConversion, Template
+from . import TemplateModel
+
+__all__ = [
+    "stratify",
+]
+
+
+def stratify(
+    template_model: TemplateModel,
+    *,
+    key: str,
+    strata: Set[str],
+    structure: Optional[Iterable[Tuple[str, str]]] = None,
+    directed: bool = False,
+    conversion_cls: Type[Template] = NaturalConversion,
+) -> TemplateModel:
+    """Multiplies a model into several strata.
+
+    E.g., can turn the SIR model into a two-city SIR model by splitting each concept into
+    two derived concepts, each with the context for one of the two cities
+
+    :param template_model: A template model
```

Comment:
Could we use the numpy convention here for docstirngs? I'm more familiar with that one and I think it's more readable.

Reply:
Done in 987937f. In #11, I added a few automatable checks and updated the configuration to work with napoleon style docstrings